allensdk/brain_observatory/behavior/behavior_session.py

```diff
@@ -83,6 +84,36 @@ def list_api_methods(self) -> List[Tuple[str, str]]:
         method_names = [m[0] for m in methods]
         return list(zip(method_names, docs))
 
+    def list_data_attributes_and_methods(self) -> List[str]:
+        """Convenience method for end-users to list attributes and methods
+        that can be called to access data for a BehaviorSession.
+
+        NOTE: Because BehaviorExperiment inherits from BehaviorSession
+        this method was also be available there.
```

Comment:
I think "was" should be "will" in this docstring

Reply:
Thanks for catching that, fixed.